Resources/Prototypes/_NF/Catalog/Fills/Items/briefcases.yml

```
@@ -32,7 +32,7 @@
 - type: entity
   parent: SpaceCashExpedition
   id: SpaceCashMinorExpedition
-  name: spesos (10000)
+  name: spesos (10.000)
```

Consistency with en-US cultureinfo.
```suggestion
  name: spesos (10,000)
```